targets/mbedos5/README.md

```diff
@@ -44,52 +44,32 @@ instructions above.
 make is used to automate the process of fetching dependencies, and making sure that
 mbed-cli is called with the correct arguments.
 
-### (optional) jshint
+### nodejs
 
-jshint is used to statically check your JavaScript code, as part of the build process.
-This ensures that pins you are using in your code are available on your chosen target
-platform.
+a recent version of [node.js](http://nodejs.org/).
 
-### Python modules
+### gulp
 
-There are multiple Python module dependencies which can be installed with the
-following command invoked from your jerryscript/targets/mbedos5/tools directory:
+[gulp](http://gulpjs.com/).
 
-```bash
-pip install -r requirements.txt
-```
-
-If you have no pip installed but your are on an apt-managed system (e.g.,
-Ubuntu), issue the following command first:
+([mbed-js-gulp](https://github.com/ARMmbed/mbed-js-gulp))
```

Comment:
Please add some more text here. Right now, it's just two links in two lines without any explanation. What is gulp, why is there an mbed-js variant of gulp, and why is the latter one in parentheses? (Is that not needed, actually?)